tests/comfy_cli/command/test_assets_library.py

```
@@ -105,6 +134,44 @@ def test_404_is_asset_not_found_not_workflow_not_found(self, cloud_target, monke
         assert err["details"]["hash"] == "comfyorg_logo.png"
         assert err["details"]["operation"] == "ensure"
 
+    def test_non_object_body_is_cloud_http_error(self, cloud_target, monkeypatch, capsys):
```

Pylint (4.0.7)

[convention] Missing function or method docstring

(C0116)